docs/Tutorials/BeginnersTutorial.md

```diff
@@ -0,0 +1,547 @@
+(it's just a python script so it isn't really built). Inside the `runs`
+directory where you have the `H5` files, run
+
+```
+../bin/GenerateXdmf \
+  --file-prefix ExportTimeDependentCoordinates3DVolume \
+  --subfile-name element_data --output BBH_Coords
+```
+
+The `--file-prefix` argument must be whatever is before the number `0` in the
+name of your `Volume` file. We output volume data per node so we append the
+node number to each volume file we have. Since you're most likely running on a
+laptop, you'll only be running on one node so you should only get one output
+file for the volume. The `--subfile-name` argument is the group name inside the
+`H5` file where the data is stored (groups can be checked by
+`h5ls -r FILE_NAME`). `GenerateXdmf` will generate a file called
+`BBH_Coords.xmf`. Make sure to keep this `.xmf` file next to the volume file it
+was generated from. It uses relative paths to find the volume file which means
+if you move it, you won't be able to visualize anything.
+
+Now you can open a Paraview window outside of the Docker container. We don't
+have Paraview in the container and even if we did, it would be difficult to open
+a GUI via a container. Open the `BBH_Coords.xmf` file you just generated inside
+Paraview. You may be prompted to choose which XDMF reader to use. Choose the
+`XDMF Reader` option. The `Xdmf3` options won't work. Once you choose a reader,
+on the left, you'll see
+
+\image html beginners_paraview_left.png "Paraview side-bar"
+
+You can uncheck all the boxes in the `Point Arrays` section as they aren't
+necessary for visualizing the coordinates. Then hit `Apply`. Now you should see
+a solid sphere. This isn't super helpful. In the top bar you should see a
+dropdown to change the style that the points are plotted in. Select `Surface
+With Edges` like so. (Note: Your top bar my look slightly different from this
+depending on what version of `Paraview` you have.)
+
+\image html beginners_paraview_top.png "Paraview top-bar"
+
+Now you'll have a solid sphere with highlighted lines sphere. To view the
+interior of the domain, you'll need to add a filter. Probably the two most
+helpful filters for viewing the domain are the `Slice` and `Clip` filters. (Note
+that you'll have to choose the `Surface With Edges` option for each filter
+separately.)
+
+`Slice` is fairly self explanatory in that it will show you a single plane
+through the domain. Experiment with different planes to see our whole domain
+structure!
+
+The `Clip` filter will remove all points "above" a certain plane, where "above"
+is in the direction of the normal of that plane. If you combine two orthogonal
+`Clip`s, you can actually view a 3D wedge of our domain. Try moving the centers
+of the planes to view the domain around our excision surfaces! They have a lot
+of cool structure.
+
+If you'd like to read more about our BBH domain, you can look at the
+documentation for `domain::creators::BinaryCompactObject`.
```

Comment:
Maybe add a few notes on (a subset of:) blocks, elements, LGL grid points within each element?

Reply:
I think that's a bit too technical for this tutorial. I also briefly explain the relationship between blocks and elements later in the tutorial so I'll leave this part as is.